internal/jobs/common.go

```diff
@@ -20,10 +20,22 @@ func decodeJob(ctx context.Context, job dejq.Job, argInterface interface{}) erro
 	return nil
 }
 
-func contextLogger(ctx context.Context, jobName string, args interface{}, accountId, reservationId int64) context.Context {
+func contextLogger(ctx context.Context, accountId, reservationId int64) context.Context {
+	logger := ctxval.Logger(ctx).With().
+		Int64("reservation_id", reservationId).
+		Int64("account_id", accountId).Logger()
+
+	ad := dao.GetAccountDao(ctx)
+	account, err := ad.GetById(ctx, accountId)
+	if err != nil {
+		logger.Warn().Msgf("Unable to fetch account info for: %d", accountId)
+	} else {
+		logger = logger.With().
+			Str("account_number", account.AccountNumber.String).
+			Str("org_id", account.OrgID).Logger()
+	}
+
 	newContext := ctxval.WithAccountId(ctx, accountId)
-	logger := ctxval.Logger(newContext).With().Int64("reservation_id", reservationId).Logger()
-	logger.Info().Interface("args", args).Msgf("Processing job: '%s'", jobName)
 	newContext = ctxval.WithLogger(newContext, &logger)
 	return newContext
 }
```

Comment:
Every job log now also contain account_number and org_id for better searching in Kibana.